README.md

```diff
@@ -513,7 +511,12 @@ cdnvm(){
         fi
     fi
 }
-alias cd='cdnvm'
+cd-nvm() {
+    cd "$@";
+    auto-nvm
+}
+alias cd='cd-nvm'
+auto-nvm
```

Comment:
seems like it might be simpler to call `cd $PWD` rather than introducing twice as many functions into the shell?

Reply:
You're perfectly right. I just amended it.